controllers/storageclaim_controller.go

```diff
@@ -491,6 +491,7 @@ func (r *StorageClaimReconciler) getCephRBDStorageClass(data map[string]string)
 			Name: r.storageClaim.Name,
 			Annotations: map[string]string{
 				"description": "Provides RWO Filesystem volumes, and RWO and RWX Block volumes",
+				"reclaimspace.csiaddons.openshift.io/schedule": "@weekly",
```

Comment:
We dont have support for storageclass level reclaimspace in 4.16, its only present on 4.17+

Reply:
No fix for it, the feature at SC is supported in 4.17. There are other ways to reclaim the space if required like annotating the namespace/pvc or creating the reclaimspacecron job etc.

Reply:
discussed offline, we don't back-port feature and my original assumption was, annotation as a feature on sc existed in 4.16. closing this.